agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/SnippetConfiguration.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.applicationinsights.agent.internal.configuration;
+
+import io.opentelemetry.javaagent.bootstrap.servlet.ExperimentalSnippetHolder;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+// preview Configuration
+public class SnippetConfiguration {
+  private static String snippet = readSnippet();
+
+  public static String readSnippet(){
+    try {
+      Path path = Paths.get("/resources/snippet.txt");
+      byte[] bytes = Files.readAllBytes(path);
+      return new String(bytes, Charset.defaultCharset());
+    } catch(IOException e){
+      throw new UncheckedIOException(e);
```

Comment:
this will bubble up. in this case, snippet enablement will fail.. maybe better to log an error and don't crash the app if that exception is not handled anywhere?